src/hotspot/os_cpu/bsd_aarch64/pauth_bsd_aarch64.inline.hpp

```diff
@@ -25,6 +25,8 @@
 #ifndef OS_CPU_BSD_AARCH64_PAUTH_BSD_AARCH64_INLINE_HPP
```

Comment:
Are these two files different enough to separate them for BSD and Linux?

Reply:
My motivation was to avoid having any ifdefs - but we need one anyway for the apple ifdef.

If I merged the two we would end up with just the contents of the BSD version of the file.

There is also the windows version of the file, which for now has empty functions. If PAC in windows is added, that'll either use the same code or maybe Windows will provide an API (like the Apple one). Merging everything would mean windows gains the UseROPProtection check.

Reply:
>Are these two files different enough to separate them for BSD and Linux?

Merging these files then broke everything for windows (because the asm function is different). Having a "ifdef apple, elseif windows else" doesn't really make sense, so I'll split the files out again.